packages/ui/src/models/visualization/flows/support/camel-component-schema.service.ts

```diff
@@ -495,8 +499,8 @@ export class CamelComponentSchemaService {
   }
 
   static canBeDisabled(processorName: keyof ProcessorDefinition): boolean {
-    const processorDefinition = CamelCatalogService.getComponent(CatalogKind.Pattern, processorName);
+    const processorDefinition = CamelCatalogService.getComponent(CatalogKind.Processor, processorName);
 
-    return processorDefinition?.propertiesSchema?.properties?.disabled !== undefined;
+    return Object.keys(processorDefinition?.properties ?? {}).includes('disabled');
```

Comment:
Fix for the `canBeDisabled` functionality, before we were checking the `Pattern` catalog only, now we check the `Processor` one which has all EIP + entities.